boards/common/nrf52/Makefile.include

```diff
@@ -0,0 +1,22 @@
+# this module contains shared code for all boards using the nrf52 CPU
+export CPU = nrf52
+
+# include this module into the build
+INCLUDES += -I$(RIOTBOARD)/common/nrf52/include
+USEMODULE += boards_common_nrf52
+
+# set default port depending on operating system
+PORT_LINUX ?= /dev/ttyACM0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+include $(RIOTMAKE)/tools/serial.inc.mk
+
+# setup JLink for flashing
+export JLINK_DEVICE := nrf52
+
+# special options when using SoftDevice
+ifneq (,$(filter nordic_softdevice_ble,$(USEPKG)))
+  export JLINK_PRE_FLASH := erase\nloadfile $(BINDIR)/softdevice.hex
+  export FLASH_ADDR := 0x1f000
```

Comment:
Is there a way to get this value from the ld script instead ?

Reply:
I have no idea, but then again, I'd say this is not part of this PR...